src/ert/config/ert_script.py

```diff
@@ -107,6 +107,9 @@ def initializeAndRun(
             error_msg = "Script cancelled (CTRL+C)"
             self.output_stack_trace(error=error_msg)
             return None
+        except UserWarning as uw:
```

Comment:
Does it make sense to treat the Warning as an Exception?

Reply:
But `UserWarning` is technically an exception? I think I need to catch this to be able to redirect the information to the dialog?

Reply:
It is strange, but probably just how Python is.